ddp-workspace/projects/ddp-atcp/src/app/services/serviceAgents/statisticsServiceAgent.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { UserServiceAgent } from 'ddp-sdk';
+
+export interface Statistic {
+  configuration: {
+    type: string;
+    questionStableId: string | null;
+    answerValue: string | null;
+  };
+  statistics: {
+    name: string;
+    data: {
+      count: string;
+      optionDetails: any;
+    };
+  }[];
+}
+
+export type StatisticsResponse = Statistic[];
+
+export enum StatisticTypes {
+  Distribution = 'DISTRIBUTION',
+  Kits = 'KITS',
+  MailingList = 'MAILING_LIST',
+  Participants = 'PARTICIPANTS',
+  SpecificAnswer = 'SPECIFIC_ANSWER',
+}
+
+@Injectable({
+  providedIn: 'root',
+})
+export class StatisticsServiceAgent extends UserServiceAgent<
```

Comment:
Can we make this service part of SDK? It might be useful for other studies.
Interfaces and types can be extracted into sdk/models directory.

Reply:
Done.